assets/js/components/HostSettings/HostSettings.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+import PageHeader from '@components/PageHeader';
+import BackButton from '@components/BackButton';
+import LoadingBox from '@components/LoadingBox';
+import { HostInfoBox } from '@components/HostDetails';
+
+import ChecksSelection from '@components/ChecksSelection/ChecksSelection';
+
+import { updateCatalog } from '@state/actions/catalog';
+import { getCatalog } from '@state/selectors/catalog';
+import { getHost } from '@state/selectors';
+
+function HostSettings() {
```

Comment:
I would be nice to have a storybook story for this. I think we should aim for that.

Reply:
I didn't add it because I believed that the stories we have for the major composing parts could be enough and because we consciously decided not to hook redux with storybook for such _pages_ top level containers. 
In fact, none of the top level pages have stories (except for AboutPage which is different).

Anyhow, if the existing stories are not enough and we want a comprehensive story about the composed page we could add an intermediate component (maybe `HostChecksSelection`? suggestions welcome) that simply gets the host and catalog from the page component, and renders everything, so we could story that one. 
How would that work?

Reply:
Well, i think many of the top level views have storybook stories, at least almost all of the new ones.
Using the `...Page` component thing. Like the `ExecutionResults` and `ExecutionResultsPage` combo (used really really often in the new components). We don't need to hook redux this way.

In fact, in my personal implementations it even helped to develop faster/better, as i'm not forced to have updated redux states while i'm just changing simple styles, etc